.github/workflows/failure_free_docs.yml

```
@@ -0,0 +1,29 @@
+name: failure_free_docs
+on:
+  pull_request:
+    paths:
+      - "docs/fail/**"
+
+permissions:
+  contents: read
+
+jobs:
+  checks_docs_for_failures:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Check PR for failures
+        run: |
+          FAILING=$(find ./docs/fail -type f)
+          if [ -z "$FAILING" ]; then
```

I think this should be able to use a bash existence check instead? (The `./docs/fail` directory isn't committed to the repo, since there is no need as rundoc creates it on demand if needed.)

```suggestion
          if [[ -e ./docs/fail ]]; then
```